tasks/main.yml

```diff
@@ -110,3 +110,6 @@
   when:
     - puppet_agent_fire_and_forget
     - puppet_lock_stat.stat.exists == false
+
+- name: Restart running services
+  command: /usr/bin/systemctl try-restart *.service
```

Comment:
should we restart blindly *every* service, or is it worth it to have a list of services that we loop over (might take longer, but I feel like less has the potential of going wrong that way)?

Reply:
other than mysql, nginx and httpd, are there other services that we `maintain' in puppet yet do not restart on configuration change?